sky/utils/schemas.py

```diff
@@ -567,7 +569,7 @@ def get_config_schema():
         # Validation may fail if $schema is included.
         if k != '$schema'
     }
-    resources_schema['properties'].pop('ports')
+    resources_schema['properties'].pop('port', None)
```

Comment:
`ports` should be the correct field?

Reply:
Yes, `ports is correct as it pops this from `_get_single_resources_schema`
https://github.com/skypilot-org/skypilot/blob/8a0a34d36bc75cd1e337e11d14e5bf8aeb4144e4/sky/utils/schemas.py#L659